File: auditHub/src/main/java/com/thy/audithub/dto/MudurlukStats.java
package com.thy.audithub.dto;

import lombok.Data;
import java.util.List;

@Data
public class MudurlukStats {
    private String              name;
    private List<DashboardEntry> monthEntries; // 12 eleman, veri yoksa null
    private DashboardEntry      average;       // hesaplanmış ortalama, hiç veri yoksa null
    private double              formatUygunPercent;    // ortalamada Format Uygunluk yüzdesi
    private double              icerikUygunPercent;    // ortalamada İçerik Olgunluk yüzdesi
}

File: Models/MudurlukStats.cs
namespace AuditHub.Models;

public class MudurlukStats
{
    public string Name { get; set; } = "";
    public List<DashboardEntry?> MonthEntries { get; set; } = [];
    public DashboardEntry? Average { get; set; }
    public double FormatUygunPercent { get; set; }
    public double IcerikUygunPercent { get; set; }
    public int DataCount { get; set; }
}
